### DISCUSSION TITLE
Screenshot to Custom location with format similar to Default?

### DISCUSSION THREAD

**Discussion by brentschooley** [WarmUpTill/SceneSwitcher#796] [Q&A] [answered]
I'm trying to take a screenshot of two different Scenes every 10 minutes to create two separate timelapses. At the moment I'm using the Default location to store the screenshot because I can't get the Custom location to behave the same way (saving files with timestamps) in a custom folder. Does anyone know how to make this work?

**Comment by WarmUpTill** [comment, marked as answer]
You will have to use a variable in the screenshot output path.

For an example how to assign the current date to a variable have a look at this discussion:
https://github.com/WarmUpTill/SceneSwitcher/issues/664#issuecomment-1397412648

Once you have set up the variable to your liking you can use it like this (assuming it is named `Current Date`:)
![Screenshot](https://github.com/WarmUpTill/SceneSwitcher/assets/19472752/1a6d5ed2-0d17-4f9e-b301-f08cb2d93ec8)

Let me know if you need further details! :)

**Comment by brentschooley** [comment]
Perfect, got it working. I actually had it set up correctly but it wouldn't populate the variable when I used the "Run macro" button to test it :) 

Thanks for your help and for this amazing tool!
